addons/website/views/snippets/s_banner.xml

```diff
@@ -10,7 +10,7 @@
                     <h1><font style="font-size: 62px;">Sell Online. Easily.</font></h1>
                     <p class="lead">This is a simple hero unit, a simple jumbotron-style component <br/>for calling extra attention to featured content or information.</p>
                     <p class="lead"><br/></p>
-                    <a href="#" class="btn btn-epsilon rounded-circle">Do something</a>
+                    <a href="#" class="btn btn-alpha">Do something</a>
```

Comment:
I don't think the commit message of this is the reason but I'll keep that change @Cocographique